dialogue-annotations-processor/src/test/java/com/palantir/dialogue/annotations/processor/DialogueRequestAnnotationsProcessorTest.java

```diff
@@ -97,6 +97,8 @@ private Compilation compileTestClass(Path basePath, Class<?> clazz) {
                 clazz.getSimpleName() + ".java"));
         try {
             return Compiler.javac()
+                    // This is required because this tool does not know about our gradle setting.
```

Comment:
I thought this used the runtime version which executes the test?

Reply:
I have java 15 locally, and it was giving me grief:

```
Expected 0 warnings, but found the following 1 warnings:
Supported source version 'RELEASE_11' from annotation processor 'com.palantir.dialogue.annotations.processor.DialogueRequestAnnotationsProcessor' less than -source '15'

Expected 0 warnings, but found the following 1 warnings:
Supported source version 'RELEASE_11' from annotation processor 'com.palantir.dialogue.annotations.processor.DialogueRequestAnnotationsProcessor' less than -source '15'

	at com.palantir.dialogue.annotations.processor.DialogueRequestAnnotationsProcessorTest.assertTestFileCompileAndMatches(DialogueRequestAnnotationsProcessorTest.java:85)
	at com.palantir.dialogue.annotations.processor.DialogueRequestAnnotationsProcessorTest.testExampleFileCompiles(DialogueRequestAnnotationsProcessorTest.java:52)
	at [[Reflective call: 4 frames collapsed (https://goo.gl/aH3UyP)]].(:0)
	at [[Testing framework: 49 frames collapsed (https://goo.gl/aH3UyP)]].(:0)
	at java.base/java.util.concurrent.RecursiveAction.exec(RecursiveAction.java:189)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:290)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1016)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1665)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1598)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:183)
```

and I like running tests locally...